js/main.js

```diff
@@ -1,2 +1,45 @@
 /* global data */
 /* exported data */
+
+var $urlInput = document.querySelector('.url');
+var $imageEntry = document.querySelector('.image-entry');
+var $entryForm = document.querySelector('#journal-entry-form');
+
+function addPhoto(event) {
+  // console.log(urlInput.value);
+  $imageEntry.setAttribute('src', $urlInput.value);
+
+}
+
+$urlInput.addEventListener('input', addPhoto);
+
+function submitInfo(event) {
+  // event.preventDefault();
+  var formInfoObj = {};
+  // console.log('Submitted');
+  // console.log($entryForm.elements);
```

Comment:
Please remove all commented out code. Also, you will need `event.preventDefault()` to keep the whole page from reloading.